moped-database/migrations/1699568640283_squashed-migration-add-audit-field-to-proj-components/up.sql

```diff
@@ -0,0 +1,22 @@
+alter table "public"."moped_proj_components" rename column "date_added" to "created_at";
+
+alter table "public"."moped_proj_components" add column "created_by_user_id" integer null;
+
+alter table "public"."moped_proj_components" add column "updated_by_user_id" integer null;
+
+alter table "public"."moped_proj_components" add column "updated_at" timestamptz null;
+
+CREATE OR REPLACE FUNCTION update_updated_at_column()
```

Comment:
i do not feel strongly about this at all, but i guess i prefer `set_updated_at_column()` since that reads a little more cleanly. i also think it's nice for the trigger name and function to be similar. trigger name is great, by the way

Reply:
Yea John, I totally agree with you. i'm not sure where that came from, but I'm furthest from liking it and even thought for a bit what would be better and didn't have anything come to mind. `st_updated_at_column()` is perfect. It says what it does and doesn't re-use words back to back which is a semantic disaster. I'll update this for sure. Thank you!

Reply:
I am going to use the existing [`public.set_updated_at()`](https://github.com/cityofaustin/atd-moped/blame/b96dea54a11544dc07a9d2eee40f0444087f3151/moped-database/migrations/1686243614837_create_work_auth_table/up.sql#L87-L93) function. John wrote that in 4 months ago, and it is a better version of what I initially proposed in this PR because mine turns out had a assignment vs. comparison (`:=` vs. `=`) operator mix-up. 🙈 John's is perfect for this task.

Reply:
oh, cool—that works too! i didn't realize how they were basically the exact same function. glad you caught the assignment bug!